sympy/stats/random_matrix.py

```diff
@@ -0,0 +1,53 @@
+from sympy import Basic, Symbol, Tuple, Expr
+from sympy.matrices.immutable import ImmutableMatrix
+from sympy.stats.rv import RandomSymbol
+from sympy.stats.index_rv import IndexedRandomSymbol
+from sympy.core.compatibility import string_types
+
+class RandomMatrix(ImmutableMatrix):
```

Comment:
I would not add this class. I would just use `ImmutableMatrix` and handle it separately. I suggest to focus on subclass `MatrixSymbol` instead.